meta_jetpack5.json

```diff
@@ -1,8 +1,9 @@
 {
-  "module_id": "viam:mlmodelservice-triton-jetpack",
+  "$schema": "https://dl.viam.dev/module.schema.json",
+  "module_id": "viam:mlmodelservice-triton-jetpack5",
```

Comment:
That was my intention, though I'm open to disagreement if you want something different. I'd like to avoid the issues we've had with the `pi` board model being for a raspberry pi 4 but not the raspberry pi 5. By giving them all numbers, it's clear which one is Jetpack 5 and which is Jetpack 6, and there is no ambiguity about a numberless Jetpack.

Reply:
Abe is working on extending the module tags so that a single module can support jetson5, jetson6, and cuda. but that's not ready yet. When it is ready, I can envision a world with the opposite scenario, where the three new modules are EOL and people need to switch back to a more general `mlmodelservice-triton` with the right tags to get new features. but that depends on Abe's work, and won't be ready for a while.

Reply:
Alright, I'm pretty OK with it.